python/ml4ir/applications/ranking/model/metrics/metrics_helper.py

```diff
@@ -5,12 +5,12 @@
 
 # Metrics where higher value is better/desirable
 POSITIVE_METRIC_SUFFIXES = [
-    "MRR",
-    "NDCG_mean"
+    "MRR"
```

Comment:
I'm not sure to understand what the changes here have to do with the PR, but 1 - NDCG, could be named inverse NDCG , or reciprocal dcg, i'm not sure intrinsic_failure is very indicative

Reply:
I will remove these changes. I do not remember why they are here either.